modules/series-service-api/src/main/java/org/opencastproject/series/api/Series.java

```diff
@@ -0,0 +1,107 @@
+/**
+ * Licensed to The Apereo Foundation under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ *
+ * The Apereo Foundation licenses this file to you under the Educational
+ * Community License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License
+ * at:
+ *
+ *   http://opensource.org/licenses/ecl2.txt
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ */
+
+package org.opencastproject.series.api;
+
+import org.opencastproject.metadata.dublincore.DublinCoreCatalog;
+
+import java.util.Date;
+
+
+/**
+ * An Opencast series.
+ */
+public class Series {
+  /** The ID of this series */
+  private String id;
+
+  /** The organization this series belongs to */
+  private String organization;
+
+  /** Serialized dublin core metadata catalogue */
+  private DublinCoreCatalog dublinCore;
+
+  /** Serialized access control lists */
+  private String accessControl;
+
+  /** Date of the last time anything about this series was modified */
+  private Date modifiedDate;
+
+  /**
+   * Date of the last time this series was deleted, or {@code null} if it is not currently deleted.
+   */
+  private Date deletionDate;
+
+
+  public String getId() {
+    return this.id;
+  }
+
+  public void setId(String id) {
+    this.id = id;
+  }
```

Comment:
Is there precedence on these builder objects in OC? Is that something Java-people do often? If you think it would be better, I can change it. But it doesn't really benefit us here. 

I'm way more interested in just making all fields of the class public and getting rid of all methods entirely. But checkstyle won't allow it. Planning on starting a discussion about this in the matrix channel, once I find the energy to do so :P

Reply:
- If I remember correctly, the Apache and Elastic modules or the AssetM;anager do that and it's essentially how Java's Streams work
- Most old code would probably expect a constructor with arguments to allow creating an object quickly without a lot of setter calls
- I'm a Python dev in heart → everything should be public. Don't know why anyone ever would want to make things private ;-P

Reply:
Okay interesting. 

> Not that important

So then lets leave it for now. I can still return later and change stuff.